backend/ShoulderToShoulder/s2s/db_models/event.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
```

Comment:
For documentation purposes, can you add doc strings before each of the models describing what they are used to represent?